### DISCUSSION TITLE
Differences betwen minemacs and doom?

### DISCUSSION THREAD

**Discussion by azzamsa** [abougouffa/minemacs#44] [General]
Hi.

Would you like to put list of the differences between mineemacs and doom?
One thing I find is that even if `me-evil` enabled, I don't have [ranger mode](https://github.com/doomemacs/doomemacs/tree/4e105a95af9c4c7e86471e5566eb7a5ff776ec92/modules/emacs/dired#module-flags) enabled in dired.

**Comment by abougouffa** [comment]
There are many!

MinEmacs is not intended to be a Doom clone, it is just my personal Emacs config, and it happened that I was using Doom before and got used to some of its bindings. So I've used some similar key bindings and features I was familiar with in Doom.

The packages used by Doom and these used by MinEmacs are not the same. I, personally, don't use `ranger-mode`, but `dirvish`, which can be enabled via the `me-files` module.
